Localization/de-DE/home.yaml

```
@@ -41,6 +53,49 @@ Für technische Hilfe: ",
 #URLs - Facebook and Twitter have localised name in certain languages:
 "foldingforum.org",
 "Facebook",
-"Twitter"
+"Twitter",
+
+#LATEST POSTS bold text
+#"LATEST POSTS",
+"NEUESTE ARTIKEL",
```

im personally unsure if artikel is the right word here. i think that posts is something else than what you think of if you say "artikel" in german. i think we could keep "posts". its more or less accepted in german too (duden has it: https://www.duden.de/rechtschreibung/Post.

```suggestion
"NEUESTE POSTS",
```

---

agree

---

Technically I agree, but the link on the web page doesn't go to the forums but to the pages where news are published. Because of the "news" character, and the length of the "posts", I went with Artikel.

---

I think we should stick with the english one, cause articles are much longer as in newspaper articles.